src/SwedbankPay/Api/Service/Payment/Resource/Request/Metadata.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SwedbankPay\Api\Service\Payment\Resource\Request;
+
+use SwedbankPay\Api\Service\Payment\Resource\Request\Data\MetadataInterface;
+use SwedbankPay\Api\Service\Resource;
+
+/**
+ * Payment metadata data object
+ */
+class Metadata extends Resource implements MetadataInterface
+{
+    /**
+     * Get Data.
+     *
+     * @param string $key
+     * @return mixed
+     */
+    public function getData($key)
+    {
+        return $this->offsetGet($key);
+    }
+
+    /**
+     * Set Data.
+     *
+     * @param string $key
+     * @param mixed $value
+     * @return $this
+     */
+    public function setData($key, $value)
+    {
+        return $this->offsetSet($key, $value);
+    }
+
+    /**
+     * Unset Data.
+     *
+     * @param string $key
+     * @param mixed $value
+     * @return $this
+     */
+    public function unsData($key)
```

Comment:
Please use complete words if possible. Also, please add a test for this method.

```suggestion
    public function unsetData($key)
```